src/translations/nb/publication.json

```diff
@@ -4,14 +4,20 @@
     "info": "Den antologien kapittelet er publisert i må være registrert først"
   },
   "contributors": {
+    "add_affiliation": "Legg til tilknytning",
     "add_author": "Legg til forfatter",
     "add_authors": "Legg til flere forfattere",
     "add_other_contributor": "Legg til flere",
     "authors": "Forfattere",
+    "confirm_remove_affiliation_text": "Er du sikker på at du ønsker du å fjerne tilknytningen til {{affiliationName}} for {{contributorName}}?",
```

Comment:
setningsoppbygging 👀

Reply:
haha, klinger greit for meg ass 😄 
`Er du sikker på at du ønsker du å fjerne {{contributorName}}s tilknytning til {{affiliationName}}?`?

Reply:
det er en ekstra "du" der 😝

Reply:
haha, dæven den var godt gjemt!